src/Apps/MyCollection/Routes/EditArtwork/Components/ArtistAvatar.tsx

```diff
@@ -0,0 +1,31 @@
+import { Avatar, Flex, Text } from "@artsy/palette"
+import { Artist } from "Apps/MyCollection/Routes/EditArtwork/Utils/artworkModel"
+
+export interface ArtistAvatarProps {
+  artist: Artist
+}
+
+export const ArtistAvatar: React.FC<ArtistAvatarProps> = ({
```

Comment:
**thought:** have you considered using https://palette.artsy.net/elements/entityheader/#gatsby-focus-wrapper

Reply:
Great idea! I'm using `EntityHeader` and `EntityHeaderArtistFragmentContainer` now 👍